skin.json

```diff
@@ -18,8 +18,8 @@
 				"responsive": true,
 				"templateDirectory": "skins/WMAU/templates",
 				"messages": [
-					"wmau-talk-link",
```

Comment:
That makes sense about these not being used, thanks! I did think I'd fix it up so that they were being output in the template rather than in PHP. But that can happen separately. :)